hw1/01-fizzbuzz.js

```diff
@@ -1,16 +1,30 @@
+/* eslint-disable no-console */
 /** Exercise 01 - Fizzbuzz
 
-Write a program that writes all the numbers from 1 to 100, with some exceptions: 
-- For numbers divisible by 3, print “fizz” 
-- For numbers divisible by 5 (but not 3), print “buzz” 
+Write a program that writes all the numbers from 1 to 100, with some exceptions:
+- For numbers divisible by 3, print “fizz”
+- For numbers divisible by 5 (but not 3), print “buzz”
 - For numbers divisible by 3 and 5, print “fizzbuzz”
 
 Use console.log() to write the proper output to the command line.
 
-**/
+* */
 
 const fizzbuzz = () => {
-  // Add your code here
+  let i = 1;
+  let toPrint = '';
```

Comment:
Can you share the rationale for creating a new variable to capture the result and print? It makes sense since you are only every calling console.log at the end of the function, but why not print the result in the conditional statements instead? 

I didn't see anything in my quick review of the style guide. However, thanks for reminding me that i should declare 'i' outside of the for loop. Thanks! this is fun 😄

Reply:
Yea there is nothing in the style guide that I could find to say to do that.  It was just my personal preference to save the response like this.  Thank you for the input 😄